app/api/utils.py

```diff
@@ -104,6 +104,14 @@ def get_all_pieces(room_id):
     """
     return sorted(rooms[room_id]['bag'].items(), key=lambda x: x[0])
 
+def not_all_consonants(pieces):
+    """
+Prevents all consonants
+    """
+ while not_all_consonants('A','E','I','O','U'):
```

Comment:
Uhm...Hi, again, @kcdark. I understand what you're trying to do, but it seems your implementation is a little off. The function you've just written will recur forever because the first thing you're doing in the function is call itself. Plus your indentation is off (the `while` after the docstring) and Python will complain. If you're having trouble, I can still merge it and fix it myself. Let me know what you think.

Reply:
K. I never added the break statement. The indentation was modified when I converted it. Sorry for all the trouble. Feel free to merge and fix.

Reply:
Feel free to merge and fix.

Reply:
Alright